lib/web/apiserver.go

```diff
@@ -1772,6 +1777,46 @@ func (h *Handler) changeUserAuthentication(w http.ResponseWriter, r *http.Reques
 	}, nil
 }
 
+func (h *Handler) trySettingConnectorNameToPasswordless(ctx *SessionContext, r *http.Request, req changeUserAuthenticationRequest) error {
```

Comment:
If you only need the `*http.Request` to get the context from it, probably best to just pass in the context alone (and rename the `*SessionContext` appropriately so both aren't called `ctx`)

Reply:
done in 5fa5c0f12d7e745fd13d2018af2808dcaa907b41